state_of_html.yml

```diff
@@ -0,0 +1,28 @@
+locale: pt-BR
+namespace: css
+translations:
+  ###########################################################################
+  # General
+  ###########################################################################
+  # Intro for stateofhtml.com
+  - key: general.state_of_html.intro
+    # aliasFor: general.html2023.survey_intro # Can we do that?
+    t: |
+      Embora os desenvolvedores web tendam a se concentrar em JavaScript e CSS, nada do que fazemos seria possível sem o HTML atuando como base.
+
+      Durante muito tempo parecia que o HTML não estava evoluindo, mas as coisas podem estar mudando.
+      Novos elementos como `<selectlist>` estão no horizonte, novos recursos interessantes como popovers, e uma série de APIs do navegador relacionadas (Web Components, PWAs, etc.).
+
+      Este ano, [Lea Verou](https://lea.verou.me/) assumiu a formidável tarefa de liderar o design desta nova pesquisa, desde o conteúdo à UX e muito mais.
+
+      Com base no sucesso do [Estado do JS](http://stateofjs.com/) e do [Estado do CSS](https://stateofcss.com/), apresentamos o **Estado do HTML**;
+      a última peça que faltava para completar a trilogia, para que possamos acompanhar a evolução da plataforma web como um todo.
+
+      Benefícios para você:
+      - Os resultados da pesquisa são **usados por navegadores** e grupos de padrões **para priorização de roteiros**.
+      Suas respostas podem ajudar a implementar recursos de seu interesse, priorizar incompatibilidades de navegador e resolver lacunas na plataforma.
```

Comment:
Hi, @romulo1984 
First, congrats for the translation.
I just joined the group and would like to contribute to your PR.

The only thing I believe we can improve here is the use of the word "recursos", I think a more accurate translation would be "funcionalidades", and this way we don't repeat the word in different contexts (I'll give an example in a following comment).

So this sentence would be: "Suas respostas podem ajudar a implementar funcionalidades de seu interesse, priorizar incompatibilidades de navegador e resolver lacunas na plataforma."

Reply:
Thanks for the suggestions, @vilaboim. I pushed the recommended changes.